.gitignore

```diff
@@ -1,6 +1,6 @@
-# directories checked out by manage_externals, and other files created
-# by manage_externals
-manage_externals.log
+# directories checked out by git-fleximod, and other files created
+# by git-fleximod
+git-fleximod.log
 ccs_config
```

Comment:
All of these should be removed - this is already done in my branch.

Reply:
Restoring file to how it was.